inst/templates/ad_advs.R

```diff
@@ -0,0 +1,552 @@
+# Name: ADVS
+#
+# Label: Vital Signs Analysis Dataset for Pediatric Trials
+#
+# Input: adsl, vs
+#        WHO_bmi_for_age_boys, WHO_bmi_for_age_girls, cdc_bmiage,
+#        who_lgth_ht_for_age_boys, who_lgth_ht_for_age_girls, cdc_htage,
+#        who_wt_for_age_boys, who_wt_for_age_girls, cdc_wtage,
+#        who_hc_for_age_boys, who_hc_for_age_girls,
+#        who_wt_for_ht_boys, who_wt_for_ht_girls, who_wt_for_lgth_boys, who_wt_for_lgth_girls
+
+library(admiral)
```

Comment:
@rossfarrugia if we add library(admiralpeds), only vs_peds data is getting updated with below USUBJIDs; however adsl_peds has different set of subjects. Due to this, we are unable to merge vs_peds and ADSL_peds. Please refer below snapshots and let us know if ADSL_peds data will also get updated. 

**vs_peds:**
<img width="328" alt="image" src="https://github.com/pharmaverse/admiralpeds/assets/157117024/f626baf7-cb26-405c-b6b6-4ee2ee7811f7">

**ADSL_peds:**
<img width="274" alt="image" src="https://github.com/pharmaverse/admiralpeds/assets/157117024/75540bb8-ef79-400c-ac3a-1140296bc7c4">

Reply:
p.s. you don't need to run `library(admiralpeds)` yourself as you can use `devtools::load_all()` here as you're working within this package. the `library(admiralpeds)` is for our users so that they can first load the package and get access to all our new functions/test data/metadata/etc